channel/cmd/channel/subcmd/memory/protocol.go

```diff
@@ -0,0 +1,49 @@
+/*
+	Copyright NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package underlay
+
+import (
+	"github.com/openziti/foundation/channel2"
```

Comment:
Seems odd to import channel2 into channel(1)

Reply:
There are a few other chanel2 imports in other files. They seem to be clustered around the cmd's.

Reply:
fixed

Reply:
The cmd stuff isn't really used. Have to dig into it in the next pass and see if we want to keep it or not.